nfadeeva/assignment2/Assignment_2_Q1.py

```diff
@@ -0,0 +1,65 @@
+import unittest
+
+
+class Node:
+    def __init__(self, data):
+        self.data = data
+        self.right = None
+        self.left = None
+
+
+def find_all_ancestors(root, key, result=None):
+        """
+        Prints all the ancestors of the given key
+
+        :return list of ancestors of the given key or False if the key is not in the Tree
```

Comment:
Can you raise a KeyError instead of returning False when the key is not in the tree?

Reply:
I don't have any idea how to do it because I'm looking for the given keys recursively and I can't raise an error cause my program will fail

Reply:
Ok, let's keep it like that then!